packages/core/src/Renderer.ts

```diff
@@ -401,32 +422,32 @@ export class Renderer extends AbstractRenderer
      * @param {number} screenWidth - The new width of the screen.
      * @param {number} screenHeight - The new height of the screen.
      */
-    resize(screenWidth, screenHeight)
+    resize(screenWidth: number, screenHeight: number): void
     {
         super.resize(screenWidth, screenHeight);
 
-        this.runners.resize.run(screenWidth, screenHeight);
+        this.runners.resize.emit(screenWidth, screenHeight);
     }
 
     /**
      * Resets the WebGL state so you can render things however you fancy!
      *
      * @return {PIXI.Renderer} Returns itself.
      */
-    reset()
+    reset(): this
     {
-        this.runners.reset.run();
+        this.runners.reset.emit();
 
         return this;
     }
 
     /**
      * Clear the frame buffer
      */
-    clear()
+    clear(): void
     {
-        this.framebuffer.bind();
-        this.framebuffer.clear();
+        this.renderTexture.bind();
```

Comment:
Just wanted to check this was intentional as it changed from what the function says it will do?

Reply:
yes, its not used inside @core, but by the meaning of the function there should be renderTexture usage, not framebuffer, because framebuffer does not containt clear color.